storage/rocksdb/event_listener.h

```diff
@@ -18,9 +18,6 @@
 /* RocksDB header files */
 #include "rocksdb/listener.h"
 
-/* MyRocks header files */
-#include "./rdb_global.h"
```

Comment:
you may still need this header file if compile with -DROCKSDB_CUSTOM_NAMESPACE=my_rocksdb